src/specifications/profile-resolution/profile-resolution-specml-requirements.xspec

```diff
@@ -402,7 +402,7 @@
                pending="[dev]"/>
    <x:scenario label="9.2 Metadata Resolution | If any of the above OSCAL versions (imported document versions, source profile version, tool version) are of a different major version (the first digit differs), the tool SHOULD provide an error and cease processing."
                pending="[dev]"/>
-   <x:scenario label="9.2 Metadata Resolution | The value of metadata:last-modified in the target MUST be set with a valid timestamp representing the time the profile resolution completed."
+   <x:scenario label="9.2 Metadata Resolution | The value of metadata:last-modified in the target MUST be set with a valid timestamp. This value SHOULD represent the time the profile resolution was initiated."
```

Comment:
Actually I agree with this. (Just didn't want to be the persnickety one. Thanks AJ!)

I would even suggest "the time that resolution has occurred or that a serialized resolution (artifact) has been created" to make it even more general. (Feel free to reword that if I am using any terms we want to avoid.)